vc_zoom/indico_vc_zoom/plugin.py

```diff
@@ -516,34 +519,21 @@ def _merge_users(self, target, source, **kwargs):
     def get_notification_cc_list(self, action, vc_room, event):
         return {principal_from_identifier(vc_room.data['host']).email}
 
+    def _check_meetings(self, sender, obj, **kwargs):
+        zoom_rooms = [room.vc_room for room in obj.vc_room_associations if room.vc_room.type == 'zoom']
+        log_entry = obj.log(EventLogRealm.event, LogKind.change, 'Videoconference', 'Schedule updated', data={
+            'Meetings': ','.join([f'{room.name} (ID: {room.id})' for room in zoom_rooms]),
+            'Date': obj.start_dt.isoformat(),
+            'State': 'pending'
+        })
+        db.session.commit()
```

Comment:
doing commit here seem rather dangerous! If this is because of the log entry, you could trigger the celery task after the request:

```py
@after_this_request
def _launch_task(response):
    refresh_meetings.delay(zoom_rooms, obj.start_dt, log_entry)
    return response
```

Reply:
Sounds good